curriculum/challenges/english/25-front-end-development/quiz-css-variables/66ed9018f45ce3ece4053eb9.md

```diff
@@ -17,439 +17,445 @@ Answer all of the questions below correctly to pass the quiz.
+Custom properties are automatically converted to `px` units
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Why is it important to use fallback values when applying custom properties in CSS?
 
 #### --distractors--
 
-Placeholder distractor 1
+To avoid undefined values in unsupported browsers
 
 ---
 
-Placeholder distractor 2
+To improve performance
 
 ---
 
-Placeholder distractor 3
+To optimize rendering time
 
 #### --answer--
 
-Placeholder answer
+To reduce code complexity
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+When defining a variable `--foreground` inside a media query, what happens when the media query no longer matches the current viewport?
 
 #### --distractors--
 
-Placeholder distractor 1
+The custom property is preserved and continues to apply
 
 ---
 
-Placeholder distractor 2
+The custom property reverts to its initial value
 
 ---
 
-Placeholder distractor 3
+The custom property is recalculated based on the viewport
 
 #### --answer--
 
-Placeholder answer
+The custom property is no longer available as it is scoped to the media query
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is not true when defining custom properties?
 
 #### --distractors--
 
-Placeholder distractor 1
+Custom properties can be inherited by child elements
 
 ---
 
-Placeholder distractor 2
+Custom properties can be defined at any level in the DOM hierarchy
 
 ---
 
-Placeholder distractor 3
+Custom properties can be updated dynamically using JavaScript
 
 #### --answer--
 
-Placeholder answer
+Custom properties can override user agent styles by default
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How would the following declaration behave: `color: var(--main-color, var(--fallback-color, #000));`?
 
 #### --distractors--
 
-Placeholder distractor 1
+It will only apply `--main-color` if it's defined
 
 ---
 
-Placeholder distractor 2
+It will apply `--fallback-color` if `--main-color` is not defined
 
 ---
 
-Placeholder distractor 3
+It will default to `#000` if neither variable is defined
 
 #### --answer--
 
-Placeholder answer
+It applies `--main-color` if defined; otherwise, checks `--fallback-color;` and if both are undefined, it uses `#000`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the primary use case for the `@property` rule when working with CSS animations?
 
 #### --distractors--
 
-Placeholder distractor 1
+To modify keyframe sequences based on variable values
 
 ---
 
-Placeholder distractor 2
+To control custom property inheritance during animation
 
 ---
 
-Placeholder distractor 3
+To ensure that custom properties maintain compatibility with media queries
 
 #### --answer--
 
-Placeholder answer
+To define which custom properties are allowed to animate and set their initial values
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Why is it important to use fallback values when applying custom properties in CSS?
```

Comment:
There's already identical question used.

Reply:
resolved in new commit (changed question)